### PR TITLE
Kubectl via toolchains (documentation)

### PR DESCRIPTION
**Pull request by alex1545**
Documenting the kubectl toolchain rule usage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,7 +69,9 @@ gcloud container clusters get-credentials <CLUSTER NAME>
 ## Dependencies
 
 The rules will require the `kubectl` tool when executing the `run` action from
-bazel. If GKE is used, also the `gcloud` sdk need to be installed.
+bazel. The `kubectl` tool is configured via a toolchain rule. Read more about the kubectl toolchain [here](toolchains/kubectl#kubectl-toolchain).
```
Comment (nlopezgi) [inline README.md]: wrap line

File: toolchains/kubectl/README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+   ```
+   or
+   ```
+   ERROR: While resolving toolchains for target <build target>: no matching toolchains found for types @io_bazel_rules_k8s//toolchains/kubectl:toolchain_type
+   ```
+   or
+   ```
+   ERROR: Analysis of target '<build target>' failed; build aborted: no such package '@local_k8s_config//': The repository could not be resolved
+   ```
+First read the official Bazel toolchain docs [here](https://docs.bazel.build/versions/master/toolchains.html) on how toolchain rules work. Then, continue reading below.
```
Comment (nlopezgi) [inline toolchains/kubectl/README.md]: wrap line

File: toolchains/kubectl/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+   ```
+   or
+   ```
+   ERROR: While resolving toolchains for target <build target>: no matching toolchains found for types @io_bazel_rules_k8s//toolchains/kubectl:toolchain_type
+   ```
+   or
+   ```
+   ERROR: Analysis of target '<build target>' failed; build aborted: no such package '@local_k8s_config//': The repository could not be resolved
+   ```
+First read the official Bazel toolchain docs [here](https://docs.bazel.build/versions/master/toolchains.html)
```
Comment (nlopezgi) [inline toolchains/kubectl/README.md]: wrap this line

File: toolchains/kubectl/README.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+on how toolchain rules work. Then, continue reading below.
+
+## How to use the kubectl Toolchain
+Register the toolchains exported by this repository in your WORKSPACE and add a `kubectl_configure` target called "local_k8s_config".
```
Comment (nlopezgi) [inline toolchains/kubectl/README.md]: wrap this line

**Comment by chrislovecnm** [comment]
@alex1545 thanks! 

@nlopezgi is this feature important enough to have an small example?  Might be something I can do so that I learn more about bazel.  Also it may give us the capability to test the tool chain at the same time.

**Pull request merged**